test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -3,8 +3,10 @@
 class RemoteCheckoutV2Test < Test::Unit::TestCase
   def setup
     gateway_fixtures = fixtures(:checkout_v2)
+    gateway_api_fixtures = fixtures(:checkout_v2_api)
     @gateway = CheckoutV2Gateway.new(secret_key: gateway_fixtures[:secret_key])
     @gateway_oauth = CheckoutV2Gateway.new({ client_id: gateway_fixtures[:client_id], client_secret: gateway_fixtures[:client_secret] })
+    @gateway_api = CheckoutV2Gateway.new(secret_key: gateway_api_fixtures[:secret_key], public_key: gateway_api_fixtures[:public_key])
```

Comment:
Is there a particular reason we need to use a different gateway setup from the existing one?

Reply:
The value in @gateway_api for the secret_key and the public_key are different from the @gateway, the same secret_key used in the @gateway is not going to work in the transactions that @gatway_api works and the @gateway_api secret_key is not going to work in the transactions that @gateway does.